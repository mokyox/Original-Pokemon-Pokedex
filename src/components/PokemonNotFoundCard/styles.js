import styled from "styled-components";
import { device } from "../../styles/media";

export const EmojiWrapper = styled.span`
  font-size: 2rem;
`;

export const NotFoundCard = styled.div`
  grid-column: 2;
  margin: 0 auto;
  max-width: 75%;

  @media ${device.tablet} {
    max-width: 100%;
    padding: 0.5rem;
  }

  @media ${device.laptop} {
    grid-column-start: 2;
    grid-column-end: 4;
  }
`;
