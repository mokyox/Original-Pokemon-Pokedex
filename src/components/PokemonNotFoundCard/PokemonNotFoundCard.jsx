import { EmojiWrapper, NotFoundCard } from "./styles";

const PokemonNotFoundCard = () => {
  return (
    <NotFoundCard className="card bg-danger">
      <div className="card-body">
        <h3 className="card-title">
          <EmojiWrapper>
            <span role="img" aria-label="Scream">
              ðŸ˜±
            </span>
          </EmojiWrapper>
        </h3>
        <h3 className="card-text">
          No Pokemon found. Please refine your search
        </h3>
      </div>
    </NotFoundCard>
  );
};

export default PokemonNotFoundCard;
