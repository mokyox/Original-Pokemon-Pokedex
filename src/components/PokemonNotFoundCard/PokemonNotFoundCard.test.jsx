import React from "react";
import PokemonNotFoundCard from "./PokemonNotFoundCard";
import renderer from "react-test-renderer";

describe("<PokemonNotFoundCard/>", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<PokemonNotFoundCard />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
