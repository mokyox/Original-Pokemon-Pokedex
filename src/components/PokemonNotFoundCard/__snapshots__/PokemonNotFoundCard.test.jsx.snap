// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PokemonNotFoundCard/> renders correctly 1`] = `
<div
  className="sc-gsTCUz dWKMot card bg-danger"
>
  <div
    className="card-body"
  >
    <h3
      className="card-title"
    >
      <span
        className="sc-bdfBwQ hczRhm"
      >
        <span
          aria-label="Scream"
          role="img"
        >
          ðŸ˜±
        </span>
      </span>
    </h3>
    <h3
      className="card-text"
    >
      No Pokemon found. Please refine your search
    </h3>
  </div>
</div>
`;
